### PR TITLE
de-emphasize rack space cli and sdk

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5a004631c5/how-to/)

File: content/cloud-servers/cloud-servers-faq.md [hunk 1]
```diff
@@ -30,14 +30,12 @@ packages, see [Linux Spheres of Support for Dedicated and Managed Operations](/h
...
+All of our cloud servers can be configured as development stacks,
 database servers, and web servers. Apache, Lighttpd, Mongrel, and Visual
-Studio are all options.
+Studio are some options.
 
 You can use the programming language of your choice to interact with
-Cloud Servers. For help getting started, use any of the
-Software Development Kits (SDKs) documented at
-<https://developer.rackspace.com/docs/>.
+Cloud Servers by using a software development kit (SDK).
```
Comment (cyrichardson) [inline content/cloud-servers/cloud-servers-faq.md]: Maybe just end this with after Cloud Servers. 
And remove the "by using a software development kit (SDK)."

File: content/rackspace-cdn/features-of-rackspace-cdn.md [hunk 2]
```diff
@@ -30,9 +30,8 @@ get your site up and running.
 **Multiple interface options**
 
 Rackspace CDN is fully supported via the Rackspace Cloud Control Panel.
-You can also interact with Rackspace CDN via our REST API or one of our
-software development kits (SDK) in Ruby, PHP, Java, node.js, Python, or
-.NET.
+You can also interact with Rackspace CDN by using our REST API or a software
```
Comment (cyrichardson) [inline content/rackspace-cdn/features-of-rackspace-cdn.md]: Maybe just stop after REST API.   ??

File: content/retired-articles/managed-dns-faq.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ Any customer with a Rackspace Cloud account is eligible for Managed DNS *Early A
 
 #### What if I am currently using Rackspace Cloud DNS?
 
-**Warning:** If you’re a current Cloud DNS user with a workflow that depends on the Rackspace `pyrax` SDK or Ansible’s Rackspace modules, you need to upgrade to the latest version of `pyrax` (1.9.7) *before you sign up for* Managed DNS (Early Access). Upgrading will prevent the `pyrax` dependency from blocking the use of Cloud DNS.
+**Warning:** If you’re a current Cloud DNS user with a workflow that depends on a `pyrax` SDK or Ansible’s Rackspace modules, you should upgrade to the latest version of `pyrax` (1.9.7) *before you sign up for* Managed DNS (Early Access). Upgrading will prevent the `pyrax` dependency from blocking the use of Cloud DNS.
```
Comment (cyrichardson) [inline content/retired-articles/managed-dns-faq.md]: I'd remove mention of pyrax which a long-not-updated SDK from Rackspace. I'd just remove the whole warning.

File: content/sdks-for-java-ruby-python-nodejs-php-and-microsoftnet.md [hunk 4]
```diff
@@ -5,14 +5,13 @@ title: 'SDKs for Java, Ruby, Python, Node.js, PHP, and Microsoft.NET'
...
-last_modified_by: Stephanie Fillmon
-product: Cloud Files
-product_url: cloud-files
+last_modified_date: '2019-03-15'
+last_modified_by: Cat Lookabaugh
 ---
 
 You can download the language-specific software development kits (SDKs)
-for Cloud Files from the [Rackspace Developer website](http://developer.rackspace.com/#home-sdks). This site includes links to the SDKs for the following languages:
+for Cloud Files from the [Rackspace Developer website](http://developer.rackspace.com/#home-sdks).
```
Comment (cyrichardson) [inline content/sdks-for-java-ruby-python-nodejs-php-and-microsoftnet.md]: I'd just retire this article.

Comment (catlook) [reply content/sdks-for-java-ruby-python-nodejs-php-and-microsoftnet.md]: i did

Comment (cyrichardson) [reply content/sdks-for-java-ruby-python-nodejs-php-and-microsoftnet.md]: Duh... ok.

File: content/rackspace-cdn/features-of-rackspace-cdn.md (hunk 2 above)
Comment (catlook) [reply content/rackspace-cdn/features-of-rackspace-cdn.md]: oh, ok.  there are other sdks out there, but ok.

File: content/cloud-servers/cloud-servers-faq.md (hunk 1 above)
Comment (catlook) [reply content/cloud-servers/cloud-servers-faq.md]: there are other sdks availlable, but ok.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a267673eee/how-to/)

**Pull request merged**